packages/components/atoms/f-error-boundary/src/index.js

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Potentially stupid q – is this a component, or is it more of a service? (just as I kind of expect atoms to have a `.vue` style SFC)

Reply:
@ashleynolan I've updated the PR so that it now references a Vue component.